development-kit/pkg/engines/csharp/rules.go

```diff
@@ -16,13 +16,52 @@
 package csharp
 
 import (
+	engine "github.com/ZupIT/horusec-engine"
 	"github.com/ZupIT/horusec-engine/text"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/csharp/and"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/csharp/or"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/csharp/regular"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/csharp/and"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/csharp/or"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/csharp/regular"
+	"github.com/ZupIT/horusec/development-kit/pkg/utils/logger"
 )
 
-func AllRulesCsharpRegular() []text.TextRule {
+type Interface interface {
+	GetAllRules() (rules []engine.Rule)
+	GetTextUnitByRulesExt(projectPath string) ([]engine.Unit, error)
+}
+
+type Rules struct{}
+
+func NewRules() Interface {
+	return &Rules{}
+}
+
+func (r *Rules) GetAllRules() (rules []engine.Rule) {
+	for index := range allRulesCsharpAnd() {
+		rules = append(rules, allRulesCsharpAnd()[index])
```

Comment:
Change for usage item. See commentary in java/rules for more details

Reply:
Fixed